### PR TITLE
Computer Vision - Correcting endpoint references

### PR DESCRIPTION
**Pull request by wiazur**
## API Name (Kind)
 `ComputerVision`...

## Purpose
Describe the intention of the changes being proposed.
What problem does it solve or functionality does it add?

## Other Information
Add any other helpful information that may be needed here.

### CODE REVIEW

File: samples/ComputerVision/AreaOfInterest/Program.cs [hunk 1]
```diff
@@ -10,8 +10,8 @@ namespace Microsoft.Azure.CognitiveServices.Samples.ComputerVision.AreaOfInteres
 
     class Program
     {
-        public const string subscriptionKey = "<your training key here>"; //Insert your Cognitive Services subscription key here
-        public const string endpoint = "https://westus.api.cognitive.microsoft.com"; // You must use the same Azure region that you generated your subscription keys for.  Free trial subscription keys are generated in the westus region. 
+        public const string subscriptionKey = Environment.GetEnvironmentVariable("COMPUTER_VISION_SUBSCRIPTION_KEY"); //Insert your Cognitive Services subscription key here
+        public const string endpoint = Environment.GetEnvironmentVariable("COMPUTER_VISION_ENDPOINT"); // You must use the same Azure region that you generated your subscription keys for.  Free trial subscription keys are generated in the westus region.
```
Comment (PatrickFarley) [inline samples/ComputerVision/AreaOfInterest/Program.cs]: @wiazur , I think the comment on this line is one you'd want to rewrite - it mentions region

Comment (wiazur) [reply samples/ComputerVision/AreaOfInterest/Program.cs]: Thanks @PatrickFarley, did you remove that line? I see that Yuan merged it, and the merged file does not have that one. https://github.com/Azure-Samples/cognitive-services-dotnet-sdk-samples/blob/master/documentation-samples/quickstarts/ComputerVision/Program.cs

Comment (PatrickFarley) [reply samples/ComputerVision/AreaOfInterest/Program.cs]: The file you linked to is different from this one

Comment (wiazur) [reply samples/ComputerVision/AreaOfInterest/Program.cs]: Ah, you're right. Sorry about that. I changed the comments. Good eye, thanks.
https://github.com/Azure-Samples/cognitive-services-dotnet-sdk-samples/pull/144

**Pull request merged**